lib/spack/docs/build_settings.rst

```diff
@@ -162,7 +162,7 @@ architecture.
 Automatically Find External Packages
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-A user can run the :ref:`spack external find <spack-external-find>` command
+You can run the :ref:`spack external find <spack-external-find>` command
```

Comment:
This section needs a label of the form:

```
.. _cmd-spack-external-find:
```

That'll cause a link to these docs to be included in the command index.